Modules/HRV2/HRV2.cpp

```diff
@@ -0,0 +1,408 @@
+//
+// Created by Izabela Wilusz and Bartłomiej Ziółkowski on 08.12.2022.
+//
+#include "HRV2.h"
+#include <iostream>
+#include <valarray>
+#include <vector>
+#include <cmath>
+#include <numeric>
+#include <cstdlib>
+
+
+HRV2::HRV2()
```

Comment:
Przerzućmy do konstruktora fragmenty które odpowiadają za inicjalizację tzn. możemy pomyśleć o `SetPeakDistance()` i innych etapów które obecnie realizowane są poprzez wywoływanie odpowiednich metod. Przekazując argumenty do konstruktora możemy w nim wywołać napisane już metody - zmniejszy nam to objętośc kodu i zabezpieczy przed wywoływaniem metod które są zależne od parametrów w przypadku kiedy nie zostały one zainicjalizowane.